lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -3026,7 +3026,7 @@ NativeCodeGenerator::GatherCodeGenData(Js::FunctionBody *const topFunctionBody,
 #endif
 
         uint objTypeSpecFldInfoCount = objTypeSpecFldInfoList->Count();
-        jitTimeData->SetGlobalObjTypeSpecFldInfoArray(RecyclerNewArray(recycler, Field(Js::ObjTypeSpecFldInfo*), objTypeSpecFldInfoCount), objTypeSpecFldInfoCount);
+        jitTimeData->SetGlobalObjTypeSpecFldInfoArray(RecyclerNewArray(recycler, Js::ObjTypeSpecFldInfo::ObjTypeSpecFldInfoPtr, objTypeSpecFldInfoCount), objTypeSpecFldInfoCount);
```

Comment:
I see why you needed this change (and elsewhere) now. It is because in "WriteBarrierMacros.h" you changed "Field" macro to always take a second "RecyclerAllocatorWithBarrier" parameter, so after expansion it always has comma. If you revert the macro change and put the right default 2nd template parameter to WriteBarrierFieldTypeTraits, then most of the case we won't have that problem. (nit only)